### PR TITLE
Add insecurity data

### PR DESCRIPTION
**Pull request by StephenWoo**
Incorporates the output of file_to_json into the merged census data.

### CODE REVIEW

File: tests/test_insecurity.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+def test_insecurity_merge():
+    merge_ins_data('tests/resources/test_food_insecurity_merge_input.json',
+                   'tests/resources/df_merged_without_insecurity.json',
+                   'tests/output/df_merged_with_insecurity_actual.json')
+    check_output('tests/output/df_merged_with_insecurity_actual.json',
+                 'tests/resources/df_merged_with_insecurity_expected.json')
+
+
+if __name__ == '__main__':
```
Comment (brickman1444) [inline tests/test_insecurity.py:21]: Do you want to keep this? I've been running tests like "pytest tests/test_insecurity.py"

**Review by brickman1444 — CHANGES_REQUESTED** [review]

File: src/insecurity.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import os
+import sys
+import json
+import pandas as pd
+sys.path.append(os.path.abspath(''))
```
[the remark this answers is not in the document]
Comment (StephenWoo) [reply src/insecurity.py:5]: Nope. Will delete.

File: src/insecurity.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+
+def load_merged(src_path: str):
```
[the remark this answers is not in the document]
Comment (StephenWoo) [reply src/insecurity.py:8]: I'll move the description and add one for `save_merged()` as well. Return type of `json.load()` seems to be `Any`

File: src/insecurity.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    with open(src_path) as src_file:
+        return json.load(src_file)
+
+
+def save_merged(data, dst_path: str) -> None:
```
[the remark this answers is not in the document]
Comment (StephenWoo) [reply src/insecurity.py:13]: Will add.

File: tests/test_insecurity.py (hunk 1 above)
Comment (StephenWoo) [reply tests/test_insecurity.py:21]: No, this was for convenience in debugging in VS Code.

**Review by brickman1444 — APPROVED** [review]

**Pull request merged**